api-report/mocha-test-setup.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-/// <reference types="mocha" />
```

Comment:
You need to revert this change - I also get it when building locally, but for some reason CI doesn't expect it.  Very annoying.

Reply:
My local build doesn't have this. Should I manually add this to mocha-test-setup.api.md?

Reply:
Also, isn't it the other way around? Looks like CI is expecting this to be there?

Reply:
Yes, CI expects it to be there but my local build does not have it.  You'll need to add it back in.

Reply:
If you have a repro of this let me know. I've seen it but can't reproduce it.